modules/articles/client/services/models/article.client.model.js

```diff
@@ -0,0 +1,19 @@
+(function () {
```

Comment:
Sure, symantically speakijg, it's a model. However, we're treating it as a service; the $resource object is ambigious as to whether it's a model or service. And now this moves a file around, and adds a new directory to our structure. 

I just don't see the point in this change if we're not modifying the implementation here. Why even introduce it, just for the sake of making a point that its a model. 

I would rather wait, and address client-side models. We'll then be able to take part in a meaningful discussion, rather than tacking on a new idea that isn't relevant to the current PR. 

If everyone else is fine with it, then I'll support it. I wanted to at least point it out, and get feedback from others. 

@codydaig @ilanbiala @lirantal